### DISCUSSION TITLE
Review of Autocomplete

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#229] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/autocomplete.svg">

**Comment by UmairApps** [comment]
To Buggy, Fix first then release.

**Reply by Misokados** [reply]
Hello,
please describe the issue you faced, we will check it out.

**Comment by dancst** [comment]
Very useful. Will be available for other languages?

**Reply by Misokados** [reply]
Hello,
We're checking the request, I will contact you shortly.

**Reply by Misokados** [reply]
Hello,
We have added your request to internal tracksystem (61311), we're going to add other languages to the plugin.

**Comment by SilverThunderbird** [comment]
Sadly, buggy to the point of being unusable.

**Reply by Misokados** [reply]
Hello,
please describe the issue you faced, we will check it out.

**Comment by SilverThunderbird** [comment]
When the autocomplete suggestion box pops up, I can’t select a word from the list using the keyboard. The up and down arrow keys do nothing. In fact, I can’t find any key that allows me to select a word.  I can choose a word with the mouse, but when I resume typing, nothing happens. Letter keys, space bar, etc. do nothing. The only way to close the suggestion box is to hit the right arrow key. 
It’s so bad I’ve had to disable the plugin.

Using Desktop Editors v9.2.1.43 x64 in Windows 11

**Reply by Misokados** [reply]
Understood, we're checking the situation. I will contact you shortly.

**Reply by Misokados** [reply]
Hello,
We have found a match with know bug (internal tracknumber - 66124), we're working on it already. We will contact you once we have something to share.

**Reply by VladimirProkofiev52** [reply]
Hello! @SilverThunderbird Good news. We have released an updated version of the plugin where the issue you described has been fixed. Plugin version 1.1.0 is now available for installation.

**Comment by minatalat618** [comment]
Hello,

Would it be possible to add support for using Data Validation custom lists as the autocomplete source in spreadsheets?

For example, if a cell has a dropdown list created through Data Validation, the autocomplete plugin could prioritize or suggest values from that list while typing — similar to how Microsoft Office 2019 Excel behaves.

This would make data entry much faster and more consistent, especially for repetitive datasets and predefined categories.

Thank you.

my english is bad so use ai to describe what i want sorry

**Reply by Misokados** [reply]
Hello @minatalat618 
Sorry for the late reply. This very thread is dedicated to discussing the Autocomplete plugin. 
Meanwhile, it seems that you are referring to the Data Validation feature in spreadsheets. Please post your request on our discourse-based forum https://community.onlyoffice.com/
Our team will check it out.
